src/test/java/edu/kis/vh/nursery/RhymersDemo.java

```diff
@@ -30,4 +29,9 @@ public static void main(String[] args) {
 
 	}
 
+	private static DefaultCountingOutRhymer[] testRhymers(RhymersFactory factory) {
```

Comment:
Incorrect refactoring (the name does not make sense). This should remain in main and the rest of main should be extracted.